### DISCUSSION TITLE
I can't push commits in github.

### DISCUSSION THREAD

**Discussion by Credit-is-First** [community/community#106858] [New to GitHub]
My friend sent me his github source.
So I changed git origin to my new repository and made a lot of modifications there.

And I tried to push it but failed.

Please help me.

**Comment by firebending** [comment]
How are you, @Credit-is-First 
At first, please check username and email in git config.

Maybe this can help you.
https://stackoverflow.com/questions/13509293/git-fatal-could-not-read-from-remote-repository

Or here is a similar discussion.
https://github.com/orgs/community/discussions/106738

**Comment by Akash1134** [comment]
We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
